pinot-query-planner/src/main/java/org/apache/pinot/query/parser/CalciteRexExpressionParser.java

```diff
@@ -60,10 +60,10 @@ private CalciteRexExpressionParser() {
   // Relational conversion Utils
   // --------------------------------------------------------------------------
 
-  public static List<Expression> convertRexNodes(List<RexExpression> rexNodes, PinotQuery pinotQuery) {
+  public static List<Expression> convertRexNodes(List<RexExpression> rexNodes, List<Expression> selectList) {
```

Comment:
simplifying this code. It was taking in a wider object than required. Only access to the selectList was required